falcon/request_helpers.py

```diff
@@ -157,6 +157,24 @@ def write(self, data):
 
         raise IOError('Stream is not writeable')
 
+    def exhaust(self, chunk_size=64 * 1024):
+        """Exhaust the stream. This consumes all the data left until the
+        limit is reached.
+
+        Args:
+            chunk_size (int): The size for a chunk (default: 64 KB).
+                It will read the chunk until the stream is exhausted.
+        """
+        while 1:
```

Comment:
Nitpick: `while True` may be considered a bit more explicit/pythonic.

Reply:
Done